test/ntp_client_test.rb

```diff
@@ -442,6 +442,17 @@
           expect(subject.sntp_test(server)).to eql(false)
         end
 
+        it "returns false if sntp respong no UCST" do
+          output["stdout"] = "sntp 4.2.8p8@1.3265-o Fri Sep 30 15:52:10 UTC 2016 (1)\n" \
+            "195.113.144.2 no UCST response after 5 seconds\n"
```

Comment:
`output["stderr"] =""`, otherwise it would mean the test passes even if you got the regex wrong.